tapdb/universe_stats.go

```diff
@@ -112,9 +137,10 @@ func (u *UniverseStats) LogSyncEvent(ctx context.Context,
 		}
 
 		return db.InsertNewSyncEvent(ctx, NewSyncEvent{
-			EventTime:     u.clock.Now(),
-			AssetID:       uniID.AssetID[:],
-			GroupKeyXOnly: groupKeyXOnly,
+			EventTime:      u.clock.Now(),
+			EventTimestamp: u.clock.Now().UTC().Unix(),
```

Comment:
`EventTime` seems redundant now, is it?

Reply:
See comment above about non-breaking. We can clean this up for 0.3 when we're going to break things anyway.